src/bandu/agents/supervisor_agent.py

```
@@ -0,0 +1,128 @@
+        request = SendMessageRequest(
+            id=str(uuid4()), params=MessageSendParams(message=message)
+        )
+
+        # Send request
+        try:
+            response = await a2a_client.send_message(request)
+            logger.info(f"Received response from agent at {agent_url}")
+            result = response.model_dump()
+            logger.info(f"Response content: {result}")
+            # return response.message.parts[0].text if response.message.parts else "No response"
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)